homeassistant/components/scrape/config_flow.py

```diff
@@ -143,6 +142,49 @@ async def validate_sensor_setup(
     return {}
 
 
+async def validate_select_sensor(
+    handler: SchemaCommonFlowHandler, user_input: dict[str, Any]
+) -> dict[str, Any]:
+    """Validate sensor input."""
```

Comment:
Let's improve this doc string to explain what the function does, maybe:
```suggestion
    """Store sensor index in flow state."""
```